src/test/java/fi/otavanopisto/kuntaapi/test/ManagementPostMenuOrderMocker.java

```diff
@@ -0,0 +1,110 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.removeStub;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.junit.Assert.fail;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.tomakehurst.wiremock.client.MappingBuilder;
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder;
+import com.github.tomakehurst.wiremock.matching.UrlPattern;
+
+import fi.otavanopisto.kuntaapi.server.integrations.management.client.model.PostMenuOrder;
+
+/**
+ * Mocker that is used to mock post menu order Wordpress plugin results
+ * 
+ * @author Antti Leppä
+ */
+public class ManagementPostMenuOrderMocker extends AbstractBaseMocker {
+  
+  private static final String PATH = "/wp-json/menuorder/v1/posts/%s";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)